docs/guides/key-manager/execute-relay-transactions.md

```diff
@@ -64,9 +64,12 @@ To encode a transaction, we need the address of the Universal Profile smart cont
 import UniversalProfileContract from '@lukso/lsp-smart-contracts/artifacts/UniversalProfile.json';
 import KeyManagerContract from '@lukso/lsp-smart-contracts/artifacts/LSP6KeyManager.json';
 import { EIP191Signer } from '@lukso/eip191-signer.js';
-import { LSP6_VERSION } from '@lukso/lsp-smart-contracts/constants';
 import Web3 from 'web3';
 
+// This is the version relative to the LSP6 standard, defined as the number 6.
+// It is used as the "version specific data" within the version 0 of EIP191
```

Comment:
Please remove this line as it's not the case, the version specific data in this case is the address of the KeyManager, the lsp6 version is used within the data to sign